va/va.h

```diff
@@ -1916,6 +2016,82 @@ typedef struct _VAContextParameterUpdateBuffer
     uint32_t reserved[VA_PADDING_MEDIUM];
 } VAContextParameterUpdateBuffer;
 
+/** \brief AES CTR fullsample encryption type */
```

Comment:
Can these be made to match the existing values below?  And can you also document that these are for the attribute values for VAConfigAttribEncryption and for  encryption_type in VAEncryptionParameters?

#define VA_ENCRYPTION_TYPE_NONE 0x00000000
#define VA_ENCRYPTION_TYPE_CENC_CBC 0x00000002
#define VA_ENCRYPTION_TYPE_CENC_CTR 0x00000008
#define VA_ENCRYPTION_TYPE_CTR_128 0x00000010
#define VA_ENCRYPTION_TYPE_CBC 0x00000020

Reply:
Updated the comments.
For the values, I can't match them with the above values. Other maintainer will ask where are the holes and would like me to fill the holes.